### PR TITLE
aiohttp: update to 3.7.3

### PR DESCRIPTION
**Pull request by OriginCode**
Topic Description
-----------------

```
aiohttp => 3.7.3
yarl => 1.6.3-1
multidict => 5.1.0
```

Package(s) Affected
-------------------

aiohttp
yarl
multidict

Security Update?
----------------

No

Build Order
-----------

`multidict -> yarl -> aiohttp`

Architectural Progress
----------------------

- [ ] AMD64 `amd64`: 
- [ ] AArch64 `arm64`

Secondary Architectural Progress
--------------------------------

- [ ] Loongson 3 `loongson3`
- [ ] PowerPC 64-bit (Little Endian) `ppc64el`

Post-Merge Architectural Progress
---------------------------------

- [ ] AMD64 `amd64`: 
- [ ] AArch64 `arm64`

Post-Merge Secondary Architectural Progress
-------------------------------------------

- [ ] Loongson 3 `loongson3`
- [ ] PowerPC 64-bit (Little Endian) `ppc64el`

### CODE REVIEW

**Comment by OriginCode** [comment]
`aiohttp` FTBFS
image

File: extra-python/yarl/spec [hunk 1]
```diff
@@ -1,3 +1,4 @@
 VER=1.6.3
+REL=1
```
Comment (MingcongBai) [inline extra-python/yarl/spec]: Why?

Comment (OriginCode) [reply extra-python/yarl/spec]: yarl depends on multidict.

Comment (cthbleachbit) [reply extra-python/yarl/spec]: A direct dependency does not imply necessity to bump rel...

**Review by cthbleachbit — APPROVED** [review]

**Comment by cthbleachbit** [comment]
SHIP IT.

File: extra-python/yarl/spec (hunk 1 above)
Comment (MingcongBai) [reply extra-python/yarl/spec:2]: Please revert this change. Blocking merging for now.

Comment (MingcongBai) [inline extra-python/yarl/spec]: Please revert this change. Blocking merging for now.

**Review by MingcongBai — CHANGES_REQUESTED** [review]

**Review by MingcongBai — APPROVED** [review]
